packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -758,6 +758,8 @@ export class SnapController extends BaseController<
     StatusStates
   >;
 
+  #preinstalledSnaps: PreinstalledSnap[] = [];
```

Comment:
```suggestion
  #preinstalledSnaps: PreinstalledSnap[] | null;
```

Then assign below in the constructor